backend/routes/user.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+
+
+const baseUrl = '/user';
+
+export const userRoute = Router();
+
+//routes
+userRoute.get(baseUrl, (_req, res) => {
+    res.json({username: 'gertrud'});
+
+});
+
```

Comment:
Feels a bit wierd to include user related things when we don't have an account system in place, or maybe that's just me?

Reply:
We'll need it later so we might as well prepare